crates/control/src/visual_referee_filter.rs

```diff
@@ -65,8 +82,13 @@ impl VisualRefereeFilter {
                 })
                 .unwrap_or(Duration::from_secs(15));
             if duration_since_last_whistle.as_secs_f32() < 1.0 {
-                duration_since_last_whistle = Duration::from_secs(5)
+                duration_since_last_whistle = Duration::from_secs(8)
             }
```

Comment:
I'm not sure what this is supposed to do.
Can you find better names or leave a comment to explain why whistle detections younger than one second are replaced by ones that are eight seconds old, or why we send a whistle age of 15 seconds if we didn't hear any.

Reply:
Will be replaced by new numbers and states in a future PR, elaborating on this due to rules changes 2024.